engines/dfc_provider/app/services/supplied_product_builder.rb

```diff
@@ -22,7 +22,6 @@ def self.import(supplied_product)
       description: supplied_product.description,
       price: 0, # will be in DFC Offer
       primary_taxon: Spree::Taxon.first, # dummy value until we have a mapping
-      shipping_category: DefaultShippingCategory.find_or_create,
```

Comment:
Maybe this is why we needed to "Set default shipping_category if absent".
I think we could have instead set `shipping_category_id` to pass through to the standard variant. 

But I agree with this commit – the system has a `DefaultShippingCategory`, so we don't need to tell it what category to use.